test/e2e/artifact_test.go

```diff
@@ -573,6 +574,71 @@ var _ = Describe("Podman artifact", func() {
 		failSession.WaitWithDefaultTimeout()
 		Expect(failSession).Should(ExitWithError(125, "Error: append option is not compatible with type option"))
 	})
+
+	It("podman artifact inspect shows created date", func() {
```

Comment:
could this be part of another test, maybe one of the more basic ones?  not a requirement, just asking.